src/components/picture/index.stories.mdx

```diff
@@ -0,0 +1,145 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Picture from ".";
+import Source from "./_children/source";
+import Image from "../image";
+
+<Meta title="Components/Picture" component={Picture} />
+
+# Picture
+
+The `Picture` component and `Picture.Source` subcomponent render the HTML5 `<picture>` and `<source>` tags, respectively. Similar to the Image, the Picture component displays images as both a means of content and content support. Picture, however, allows for different images (with respective resize options) to be used explicitely for different media queries. This allows for different Art Direction approaches for specific viewport dimensions.
+
+For `resizedOptions`, here are the query parameters that are available:
```

Comment:
@JackHowa : Wanted to ask about this. This is incorrect as this param on `Picture` is currently `resizerOptions` as per the acceptance criteria on the ticket. 
> Takes in an object resizerOptions. Note: A source image token is only compatible with one source image

This is arguably confusing though as `Image` uses a (resizedOptions)[https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.0.1/src/components/image/index.jsx#L112-L113] param for the same object. I think they should both use the same param name. I like `resizerOptions` and think it's a more intuitive name but I can change it to `resizedOptions` to be consistent w/ `Image`.
cc: @nschubach

Reply:
must've been a typo on my end. Use the same prop name in the Image component -- like we've said.

Reply:
this has not been addressed. `resizedOptions` should be used in all places. we want to ensure uniformity and reusability between the image component and picture components

@ecarlisle

Reply:
I've unresolved the comment

Reply:
@JackHowa : 
Thank you! This naming convention was being used in the `format-image-resizer-src` util. I've changed this. `resizedOptions` is being used in all places.

Reply:
Updated